public/app/features/logs/components/LogDetailsRow.test.tsx

```diff
@@ -109,21 +92,22 @@ describe('LogDetailsRow', () => {
     });
 
     expect(screen.queryByTestId('logLabelStats')).not.toBeInTheDocument();
-    const adHocStatsButton = screen.getByTitle('Ad-hoc statistics');
+    const adHocStatsButton = screen.getByRole('button', { name: 'Ad-hoc statistics' });
     fireEvent.click(adHocStatsButton);
     expect(screen.getByTestId('logLabelStats')).toBeInTheDocument();
     expect(screen.getByTestId('logLabelStats')).toHaveTextContent('another value');
   });
 
-  it('should render clipboard button on hover of log row table value', async () => {
-    setup({ parsedKey: 'key', parsedValue: 'value' });
+  // TODO: skipped for now because I don't know a way of testing CSS styles
```

Comment:
If it requires testing of the presence or absense of a given CSS class or style, I would recommend to remove the test, and consider using e2e tests for this assertion if it's important. You could use a `data-` attribute for the test, but it doesn't look like it's worth the extra code.